addons/sale_project/models/project_project.py

```diff
@@ -187,7 +187,12 @@ def action_view_sos(self):
     def action_get_list_view(self):
         action = super().action_get_list_view()
         if self.allow_billable:
-            action['views'] = [(self.env.ref('sale_project.project_milestone_view_tree').id, 'tree'), (False, 'form')]
+            action['views'] = [(self.env.ref('sale_project.project_milestone_view_tree').id, view_type) if view_type == 'tree' else (view_id, view_type) for view_id, view_type in action['views']]
+        return action
+
+    def action_get_list_view_project_update(self):
+        action = super().action_get_list_view_project_update()
+        action['views'] = [(self.env.ref('sale_project.sale_project_milestone_view_tree').id, view_type) if view_type == 'tree' else (view_id, view_type) for view_id, view_type in action['views']]
```

Comment:
we should have the same condition than the one in `action_get_list_view`, no?

Reply:
TBH, I think we probably would like to have the same behavior in `action_get_list_view` and `action_get_list_view_project_update`. 🤔